src/psyclone/tests/domain/lfric/lfric_builtins_test.py

```diff
@@ -3464,20 +3719,23 @@ def test_X_innerproduct_X(tmpdir, dist_mem):
 
 
 def test_sum_X(tmpdir, dist_mem):
-    ''' Test that 1) the str method of LFRicSumXKern returns the expected
-    string and 2) we generate correct code for the built-in operation which
-    sums elements of a real-valued field X as sumfld = sum(X(:)).
+    '''Test that 1) the '__str__' method of 'LFRicSumXKern' returns the
+    expected string and 2) we generate correct code for the built-in
+    operation which sums elements of a real-valued field 'X' as
+    'sumfld = sum(X(:)). 3)'. Also test the 'metadata()' method.
```

Comment:
Quotation mark should be after `sum(X(:))` and not after `3)`. Please also remove full stop after `3)`.

Reply:
Looks like it must have been a cut-n-paste problem. Fixed now. There are no other cases of a quotation being after `3)`. I've also removed the two cases of a full stop being after `3)`.

Reply:
Also found and modified a `3):` in `lfric_integer_builtins_test.py`, but perhaps you will mention that later.